docs/pipelines/add-pipeline-step.md

```diff
@@ -12,3 +12,14 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the pipeline you want to manage and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to choose `PIPELINES`
+8. In the list of pipeline find the pipeline you want to manage and click `DETAILS`
+9. On the upper-right corner of pipeline details view click `STEPS`
+10. On the upper-right corner of the list click `ADD STEP`
+12. Follow the instructions in the add pipeline step form then click `CONFIRM`
```

Comment:
```suggestion
1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. In the list of applications find the application with the pipeline you want to manage and click `DETAILS`
1. On the upper-right corner of the applications details view click `SETTINGS`
1. Use the drop-down menu to choose `PIPELINES`
1. In the list of pipeline find the pipeline you want to manage and click `DETAILS`
1. On the upper-right corner of pipeline details view click `STEPS`
1. On the upper-right corner of the list click `ADD STEP`
1. Follow the instructions in the add pipeline step form then click `CONFIRM`
```

Cleaner option for ordered list count, avoiding to worry on item numbering

Reply:
Done, thanks!